scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -1176,6 +1190,9 @@ private void sendMessage(final RawData message) throws HandshakeException {
 
 		DTLSSession session = connection.getEstablishedSession();
 		if (session == null) {
+			if (!checkOutboundCorrelationContext(message, null)) {
```

Comment:
Future Extension.
Without that, the response is sent but then dropped by the client (also without feedback).